test/testAll.m

```diff
@@ -90,14 +90,11 @@
     nEmptyLines = 0;
     nCommentLines = 0;
 
-    nMsgs = length(checkcode(listFiles{:}));  % Check all code files at once.
-
     for i = 1:length(listFiles)
-
+        nMsgs = nMsgs + length(checkcode(listFiles{i}));
         fid = fopen(listFiles{i});
```

Comment:
Do you know, why the overall file check does not work?

Reply:
Sure.